modules/ols-creating-the-credentials-secret-using-cli.adoc

```diff
@@ -32,9 +37,36 @@ stringData:
 ----
 <1> The `apitoken` is not `base64` encoded.
 +
+.Example credential secret for {watsonx}
+[source,yaml,subs="attributes,verbatim"]
+----
+apiVersion: v1
+data:
+  apitoken: NlFy 
+kind: Secret
+metadata:
+  name: watsonx-api-keys
+  namespace: openshift-lightspeed
+type: Opaque
+----
++
 .Example credential secret for {azure-official} {openai}
+[source,yaml,subs="attributes,verbatim"]
+----
+apiVersion: v1
+data:
+  apitoken: azure_token 
+kind: Secret
+metadata:
+  name: azure-api-keys
+  namespace: openshift-lightspeed
+type: Opaque
+----
 +
-[source,yaml, subs="attributes,verbatim"]
+Alternatively, for {azure-openai} you can use {entra-id} to authenticate your LLM provider. {entra-id} users must configure the required roles for their {azure-openai} resource. For more information, see the official Microsoft link:https://learn.microsoft.com/en-us/azure/ai-services/openai/how-to/role-based-access-control#cognitive-services-openai-contributor[product documentation].
++
+.Example credential secret for {entra-id}
```

Comment:
Same issue about calling this an example while having user-replaced values:
```suggestion
.Credential secret for {entra-id}
```

Reply:
done